### PR TITLE
Updated README for the change in default config, Updated .gitignore

### PR DESCRIPTION
**Pull request by vibhoragarwal**
Updated README for change in default configurations

### CODE REVIEW

File: .idea/workspace.xml
Comment (uglow) [inline .idea/workspace.xml:1]: IDE files should be ignored by git

**Review by uglow — CHANGES_REQUESTED** [review]
The readme looks good, but the IDE file cannot be added to this repo. If you could add the file to the .gitignore, then we can merge.

File: .idea/workspace.xml
Comment (vibhoragarwal) [reply .idea/workspace.xml:1]: Yes.

**Review by uglow — APPROVED** [review]
Looks good - thanks for contributing!

**Comment by vibhoragarwal** [comment]
Thanks Brett for your plugin, very helpful indeed..

**Pull request merged**